pkg/forwarder/forwarder_health.go

```diff
@@ -106,6 +110,24 @@ func (fh *forwarderHealth) healthCheckLoop() {
 	}
 }
 
+//populates a map containing API Endpoints per API keys that belongs to the forwarderHealth struct
```

Comment:
Comment above function should starts by the function/method/attribute name (ex: https://github.com/DataDog/datadog-agent/pull/4876/files#diff-084e55fad5d2cab6ae1fe8e29acfd58cL41).